cluster/stone-prd-rh01/managed/rhtap-releng-tenant/remote-secrets/remote_secret-pyxis-redhat-acm-secret.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: RemoteSecret
```

Comment:
I do not think we need pyxis stage anymore. this file can be removed and anything referencing pyxis stage should be switched to prod.

Reply:
Done, thanks!